src/main/resources/fruits_report.csv

```diff
@@ -0,0 +1,3 @@
+fruit,quantity
+Fruit{name='banana'},1301
+Fruit{name='apple'},3023
```

Comment:
please look how this output file should look like
it should contain only fruit name